packages/adp-tooling/src/types.ts

```diff
@@ -52,6 +58,23 @@ export interface AdpWriterConfig {
     };
 }
 
+export interface FlpConfiguration {
+    title: string;
+    bspName: string;
```

Comment:
Is the BSP name part of the FLP configuration? Or is it something else?

Reply:
Moved to the CloudApp interface